src/main/webapp/app/admin/logfile/logfile.component.html

```diff
@@ -0,0 +1,13 @@
+<div>
+    <h2>View logs</h2>
+
+    <jhi-route-selector></jhi-route-selector>
+
+    <div class="to-bottom" *ngIf="logtxt" (click)="scrollToBottom()">&#8675;Bottom</div>
```

Comment:
Good idea. I've just done it.
![screenshot from 2017-08-02 16-17-04](https://user-images.githubusercontent.com/513471/28877902-11d7e136-779e-11e7-9bee-f38caa058d74.png)